R/merge_hydrofabrics.R

```diff
@@ -404,13 +432,20 @@ assign_global_terminal_identifiers = function(meta,
 
 #' @importFrom dplyr rename any_of
 renamer <- function(x) {
-  rename(x, any_of(c(id = "aggregated_ID", 
-                     id = "ID", 
-                     toid = "toID",
-                     member_comid = "member_COMID",
-                     id = "aggregated_flowpath_ID",
-                     did = "aggregated_divide_ID",
-                     levelpathid = "LevelPathID")))
+  
+  rules <- c(id = "aggregated_ID", 
+             id = "ID", 
+             toid = "toID",
+             member_comid = "member_COMID",
+             id = "aggregated_flowpath_ID",
+             did = "aggregated_divide_ID",
+             levelpathid = "LevelPathID")
+  
+  if(sum(c("reconciled_ID", "aggregated_ID", "aggregated_flowpath_ID") %in% names(x)) < 2) {
+    rules <- c(rules, id = "reconciled_ID")
+  }
+  
+  rename(x, any_of(rules))
```

Comment:
WHAT A HACK! I've kind of been dumping skeletons down here figuring we can clean up and get rid f this down the road.

Reply:
hehe 😈